packages/now-build-utils/src/detect-builders.ts

```diff
@@ -309,6 +309,13 @@ export async function detectBuilders(
     options
   );
 
+  if (frontendBuilder && framework === 'redwoodjs') {
+    // RedwoodJS uses the /api directory differently so we must
+    // clear any existing builders and only use `@vercel/redwood`.
+    builders.length = 0;
+    builders.push(frontendBuilder);
```

Comment:
Do we need to print a warning here?

Reply:
This is not really a warning, this is the expected behavior. I updated the PR description with more info.